Extension/src/LanguageServer/client.ts

```diff
@@ -915,21 +915,34 @@ export class DefaultClient implements Client {
         return (selection) ? selection.index : -1;
     }
 
-    public async handleCompilerQuickPick(): Promise<void> {
+    public async showPrompt(promptMessage: string): Promise<void> {
+        const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have IntelliSense configured. Unless you set your own configurations, IntelliSense may not be functional."), promptMessage);
+        if (value === promptMessage) {
```

Comment:
Is there ever a case when this is true?  It seems odd that we would localize the message twice (once here and once to pass it in).

Reply:
All the cases I see only pass in "Select Compiler" which won't match `value` (and `value` is never displayed in a prompt anywhere as far as I can tell)

Reply:
Perhaps I could be using the API wrong but from what I understood you want the value returned from showInformationMessage to be equal to the localized string that is passed as the second argument. In which case "value == promptMessage" will be true if the user selects the "Select Compiler" option

Reply:
I think I was confused because of the name of the variables.  I expected `promptMessage` to be the localized thing for `setCompiler.message`.  Maybe rename `promptMessage` to `buttonMessage`.